### PR TITLE
fix(docs): some wrong examples in the document

### PR DESCRIPTION
**Pull request by liunnn1994**
Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

1. Update deprecated attributes in the drawer component example
2. Fix wrong types in tree component  examples

### CODE REVIEW

File: docs/en-US/component/drawer.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ Drawer provides an API called `destroyOnClose`, which is a flag variable that in
 | ----------------- | ---------------------------------------------------------------------------------------------- |
 | —                 | Drawer's Content                                                                               |
 | header            | Drawer header section; Replacing this removes the title, but does not remove the close button. |
-| title(deprecated) | Works the same as the header slot. Use that instead.                                           |
+| title <DeprecatedTag /> | Works the same as the header slot. Use that instead.                                           |
```
Comment (chenxch) [inline docs/en-US/component/drawer.md:119]: Thanks for your contribution, `DeprecatedTag` has its own spacing, so you can remove the preceding space, and then do you mind executing `pnpm run lint:fix` again?

Comment (asdjgfr) [reply docs/en-US/component/drawer.md]: done

**Comment by chenxch** [comment]
I don't know if it's a problem with online viewing, the document seems to be unformatted, please help to confirm whether `pnpm run lint:fix` has been executed

**Comment by asdjgfr** [comment]
> @asdjgfr I don't know if it's a problem with online viewing, the document seems to be unformatted, please help to confirm whether `pnpm run lint:fix` has been executed

image
image

I have already run `lint:fix`, and no problems occurred locally.

**Comment by chenxch** [comment]
> I have already run `lint:fix`, and no problems occurred locally.

Thanks for your confirmation.

**Review by chenxch — APPROVED** [review]
LGTM.

**Review by btea — APPROVED** [review]

**Pull request merged**